.github/workflows/ci.yml

```diff
@@ -109,12 +112,7 @@ jobs:
       - name: Check Newsfragment
         if: |
           startsWith(github.ref, 'refs/pull/')
-          && !(
-            startsWith(github.head_ref, 'dependabot')
-            || startsWith(github.head_ref, 'yolo')
-            || startsWith(github.head_ref, 'release')
-            || startsWith(github.head_ref, 'revert')
-            )
+          && steps.changes.outputs.news-fragments
```

Comment:
So we use the check_newsfragments.py to check the provided newsfragement has the right format ?

Reply:
yes